client/http-client/src/transport.rs

```diff
@@ -57,13 +57,13 @@ impl HttpTransportClient {
 			return Err(Error::Url("Port number is missing in the URL".into()));
 		}
 
-		let mut connector = HttpConnector::new();
-
-		connector.set_reuse_address(true);
-		connector.set_nodelay(true);
-
 		let client = match target.scheme_str() {
 			Some("http") => {
+				let mut connector = HttpConnector::new();
+
+				connector.set_reuse_address(true);
```

Comment:
remove these to more "similar" to `HTTPs connector?

Reply:
Does removing them have any obvious impact on anything? I did a bit of reading up on them and they both sound like optimisations anyway (I wasn't sure that nodelay would actually help anything much here; it sounds like it trades a little throughput for lower latency, and reuseaddress sounds like an optimisation around freeing up an address a little quicker mainly)

Reply:
nah, no real difference might reduce the number of file descriptors for the benchmarks.

let's remove it.